.rhdh/docker/Dockerfile

```diff
@@ -234,6 +234,9 @@ RUN chmod -R a+r ./dynamic-plugins/ ./install-dynamic-plugins.py; \
 RUN mkdir /opt/app-root/src/.npm
 RUN chown -R 1001:1001 /opt/app-root/src/.npm
 
+# default npm config to the default registry, not the cachito one
+RUN npm config set registry=https://registry.npmjs.org/; npm config get registry; npm config set cafile /opt/app-root/src/registry-ca.pem
```

Comment:
I'd suggest using `&&` instead of `;` though it doesn't provide much for this sequence of commands.  I guess probably it'd be good to follow the sonarcloud advice and split this up into 3 lines.